src/SmartComponents/Actions/ViewActions.js

```diff
@@ -188,13 +195,20 @@ class ViewActions extends Component {
 }
 
 ViewActions.propTypes = {
-    match: PropTypes.any,
+    breadcrumbs: PropTypes.array,
     fetchRules: PropTypes.func,
+    history: PropTypes.object,
```

Comment:
last thing then i promise we can merge this, cleaning up the props here... we dont need history path params